ovn_cluster.sh

```diff
@@ -368,6 +368,14 @@ function start-db-cluster() {
     --ovn-northd-nb-db=${REMOTE_PROT}:${central_1_ip}:6641,${REMOTE_PROT}:${central_2_ip}:6641,${REMOTE_PROT}:${central_3_ip}:6641 \
     --ovn-northd-sb-db=${REMOTE_PROT}:${central_1_ip}:6642,${REMOTE_PROT}:${central_2_ip}:6642,${REMOTE_PROT}:${central_3_ip}:6642 --ovn-manage-ovsdb=no \
     $SSL_ARGS start_northd
+    ${RUNC_CMD} exec ${CENTRAL_NAME}-2 ${OVNCTL_PATH}  \
```

Comment:
Let's update the comment on line 366.
Should we make it more predictable on which one of the 3 northd will be
the one with the write lock? Maybe a config knob? If you do not mind,
please put a line in the commit that either points folks on knowing how
to control/show the state of the cluster.

Reply:
Done.